src/test/java/org/apache/mesos/scheduler/DefaultSchedulerTest.java

```diff
@@ -429,8 +429,8 @@ public void testOfferAcceptedOnce() throws Exception {
         defaultScheduler.resourceOffers(mockSchedulerDriver, Arrays.asList(offer));
         defaultScheduler.awaitTermination();
 
-        // Verify scheduler accepted resources only once
-        verify(mockSchedulerDriver, times(1)).acceptOffers(any(), any(), any());
+        // Verify scheduler accepted resources twice, once for scheduling and once for recovery.
+        verify(mockSchedulerDriver, times(2)).acceptOffers(any(), any(), any());
```

Comment:
@xtenzo Thanks for bringing this up. This should be fixed via: https://github.com/mesosphere/dcos-commons/pull/209

The root cause behind this was basically, two separate acceptOffer calls on the same offer in one Offer cycle. One from PlanManager and one from Resource cleaner.